app/views/groups/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="categories">
+  <div class="categories-header">
+    <%= image_tag("hamburger.png", :alt => "Menu Btn ") %>
+    <h3 class="font-bold text-4xl">CATEGORIES</h3>
+  </div>
+
+  <div id="groups" class="categories-list">
+    <% if notice.present? %>
+      <div class="notice-container">
+        <p class="notice" id="notice"><%= notice %></p>
+      </div>
+    <% end %>
+    <% if @groups.empty? %>
+      <p class="no-categories">No categories available</p>
+    <% else %>
+        <% @groups.each do |group| %>
+          <div class="category">
+            <%= link_to user_entity_path(current_user.id, group.id) do %>
+              <p class="icon">
+                <%= image_tag group.icon_url if group.icon %>
+              </p>
+              <div class="category-details">
+                <h4 class="categories-name">
+                  <%= group.name.upcase %>
+                  <%= @sum_per_category %>
+                </h4>
+                <p class="category-date-created">
+                  <%= group.created_at.to_date %>
+                </p>
+              </div>
+            <% end %>
+          </div>
+        <% end %>
+    <% end %>
+    <div class="add-category">
+      <%= link_to 'Add Category', new_user_group_path, class: "" %>
+    </div>
+  </div>
+
+
+</div>
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224415006-b4e9f60a-8a09-4b25-8575-ced72201975e.png)

**[Optional]**

- [ ] It would be great if you make your header title in the center to follow the Behance design 😊
- [ ] It would be nice if you show the total amount of transactions on their category page to follow professionalism while making your application and make your application human friendly 😊 It would be nice if you put the total amount in the blue box shown in the above image 👍
- [ ] It would be great if you make your button at the bottom of the screen with the same color green button which you have on the transactions page with an inside white border to make your application follow the Behance design 👍